tests/CaptchaActionTest.php

```diff
@@ -63,14 +64,15 @@ public function testRunRefresh()
     {
         /* @var $driver Driver|\PHPUnit_Framework_MockObject_MockObject */
         $driver = $this->getMockBuilder(Driver::class)
+            ->setConstructorArgs([$this->app])
```

Comment:
Should I really be calling `new Driver($this->app)` or is there something I'm definitely not understanding with DI ? :)

Reply:
What's the problem passing application?